.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Allow the documentation team to be informed when the PCB bucket address is updated.
+meilisearch.pkr.hcl @meilisearch/docs-team
```

```suggestion
meilisearch.pkr.hcl @meilisearch/docs-team

```

I hope this does not break though

---

😄 I don't know, I followed the GitHub documentation exactly and checked against [other repos.]
(https://github.com/meilisearch/documentation/blob/main/.github/CODEOWNERS)

---

Nop